test/cbmc/proofs/HTTPClient_Send-contracts/cbmc-viewer.json

```diff
@@ -0,0 +1,7 @@
+{ "expected-missing-functions":
```

Comment:
Do we need these log files?

Reply:
Hello, this PR contains proofs relying on experimental features of the verificaiton tool CBMC, and as such is not yet meant to be merged.

I wasn't able to tag this PR as "experimental" or "do not merge".  Would you rather have me close it for now and reopen it later ?

Reply:
Thank you for your response. Yes, we can either mark this PR as "draft" or better would be to close and reopen when you are ready.

Reply:
As you mentioned above, I am closing this PR. Feel free to open a new PR when you are ready.